docs/tutorial/nangang.md

```diff
@@ -20,7 +20,7 @@ Run two nodes and connect them:
 ./build/wakunode2 --ports-shift:0
 
 # Run another node with staticnode argument
-./build/wakunode2 --ports-shift:1 --staticnode:/ip4/0.0.0.0/tcp/60000/p2p/16Uiu2HAmF4tuht6fmna6uDqoSMgFqhUrdaVR6VQRyGr6sCpfS2jp
+./build/wakunode2 --ports-shift:1 --staticnode:/ip4/0.0.0.0/tcp/60000/p2p/16Uiu2HAmF4tuht6fmna6uDqoSMgFqhUrdaVR6VQRyGr6sCpfS2jp --storenode:/ip4/0.0.0.0/tcp/60000/p2p/16Uiu2HAmF4tuht6fmna6uDqoSMgFqhUrdaVR6VQRyGr6sCpfS2jp
```

Comment:
Could we put this in a separate self-contained docs markdown file? E.g. how to use store client in general.

Nangang is already done, so store wouldn't be part of the scope retroactively. For next testnet, we'll make sure the cluster node works as a store node and it'd make sense to have it in docs for that.